src/ui/public/courier/fetch/strategy/search.js

```diff
@@ -16,6 +16,7 @@ export default function FetchStrategyForSearch(Private, Promise, timefilter, kbn
      */
     reqsFetchParamsToBody: function (reqsFetchParams) {
       const indexToListMapping = {};
+      const timeBounds = timefilter.enabled ? timefilter.getBounds() : null;
```

Comment:
It looks like there's a [getActiveBounds](https://github.com/elastic/kibana/blob/d8d65526c6afba0e12cbc39fe844ec0a3fb9ba95/src/ui/public/timefilter/timefilter.js#L100-L100) method you could use on the timefilter.